src/cryptonote_config.h

```diff
@@ -219,6 +219,7 @@ namespace config
   const unsigned char HASH_KEY_RPC_PAYMENT_NONCE = 0x58;
   const unsigned char HASH_KEY_MEMORY = 'k';
   const unsigned char HASH_KEY_MULTISIG[] = {'M', 'u', 'l', 't' , 'i', 's', 'i', 'g', 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+  const char HASH_KEY_MESSAGE_SIGNING[] = "MessageSignature";
```

Comment:
Since this key will often be used with external systems, I would propose to change this to `MoneroMessageSignature`.

Reply:
Good idea. Will fix.